code/game/objects/items/devices/multitool.dm

```diff
@@ -96,4 +96,17 @@
 	icon = 'icons/obj/abductor.dmi'
 	icon_state = "multitool"
 	toolspeed = 0.1
-	origin_tech = "magnets=5;engineering=5;abductor=3"
\ No newline at end of file
+	origin_tech = "magnets=5;engineering=5;abductor=3"
+
+/obj/item/device/multitool/abductor/syndi

```

Comment:
It is. everything else I've done is either a numerical change or copy+pasting from elsewhere